guac_actix/Cargo.toml

```diff
@@ -9,12 +9,12 @@ actix-web = {version = "0.7",  default-features = false}
 futures = "0.1"
 failure = "0.1"
 guac_core = {path="../guac_core"}
-althea_types = {git="https://github.com/althea-mesh/althea_rs.git"}
-num256 = {git="https://github.com/althea-mesh/althea_rs.git"}
+althea_types = "0.1.0"
 log = "0.4"
 serde = "1.0.0"
 serde_derive = "1.0.0"
-serde_json = "1.0.0"
+serde_json = "1.0.24"
 qutex = "^0.2"
 tokio = "0.1"
 bytes = "0.4"
+ethereum-types = "0.3.2"
```

Comment:
this is just what's available on cargo... should be trivial to remove the git patch

Reply:
:+1: fixed. Additionally I found out `owning_ref` could be used from upstream.